apps/tsilhqotin-language-hub/src/Components/Pages/Radio/Radio.tsx

```diff
@@ -20,7 +20,7 @@ export const Radio = (): JSX.Element => {
                 <Card className="tsilhqotinRadio">
                     <img width={120} alt="radioLogo" src={getRadioConfig().radioLogoUrl} />
                     <div className="radioPlayer">
-                        <MediaPlayer audioUrl={getRadioConfig().radioAudioUrl} />
+                        <IcecastPlayer audioUrl={getRadioConfig().radioAudioUrl} />
```

Comment:
After all other refactors, check that the IcecastPlayer still works for you.

Reply:
Checked, IcecastPlayer still works.